react/features/video-menu/constants.ts

```diff
@@ -12,3 +12,25 @@ export const NATIVE_VOLUME_SLIDER_SCALE = 19;
  * recognizes whole numbers.
  */
 export const VOLUME_SLIDER_SCALE = 100;
+
+/**
+ * Participant context menu button keys.
+ */
+export const PARTICIPANT_MENU_BUTTONS = {
```

Comment:
style nit: sort variables alphabetically

Reply:
But aren't they already?

Reply:
I mean PARTICIPANT_MENU_BUTTONS relative to  VOLUME_SLIDER_SCALE